.github/workflows/knative-go-build.yaml

```diff
@@ -11,4 +11,4 @@ on:
 
 jobs:
   build:
-    uses: knative/actions/.github/workflows/reusable-go-build.yaml@main
+    uses: cardil/knative-actions/.github/workflows/reusable-go-build.yaml@feature/unit-test-each-go-module
```

Comment:
/hold

Restore when https://github.com/knative/actions/pull/223 is merged

Reply:
Looks like https://github.com/knative/actions/pull/223 helps:

- https://github.com/knative/client/pull/1953/checks?check_run_id=29579171848
- https://github.com/knative/client/actions/runs/10672089158/job/29579078173?pr=1953#step:7:49